var/spack/repos/builtin/packages/ppopen-appl-bem/package.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import os
+
+
+class PpopenApplBem(MakefilePackage):
+    """
+    ppOpen-APPL/BEM is software used to support a boundary element analysis
+    executed on a parallel computer. The current version includes a software
+    framework for a parallel BEM analysis and an H-matrix library.
+    If you want to use the framework based on dense matrix computations,
+    please move to the directory 'src/framework' and
+    'src/framework_with_template'.
+    If you want to use the H-matrix library, please
+    move to the directly 'src/HACApK_with_BEM-BB-framework_1.0.0'.
+    """
+
+    homepage = "http://ppopenhpc.cc.u-tokyo.ac.jp/ppopenhpc/"
+    url = "file://{0}/ppohBEM_0.5.0.tar.gz".format(os.getcwd())
+
+    version('0.5.0', sha256='bf5e32902c97674c99353ee35de9c89206659e82b9c3d1f6edc9beffbb7c9d5f')
+
+    depends_on('mpi')
+
+    parallel = False
+    hacapk_src_dir = join_path(
+        'HACApK_1.0.0',
+        'src',
+        'HACApK_with_BEM-BB-framework_1.0.0'
+    )
+    src_directories = [
+        join_path('bem-bb-framework_dense', 'src', 'framework_with_templates'),
+        join_path('bem-bb-framework_dense', 'src', 'framework'),
+        hacapk_src_dir
+    ]
+
+    def edit(self, spec, prefix):
+        flags = [self.compiler.openmp_flag]
+        fflags = flags
```

Comment:
I take it you're trying to have `flags` only contain the OpenMP flag and `fflags` contain everything? I don't think what you have here works, as Python will copy the pointer to the list, and modifying `fflags` will also modify `flags`. I think you want:
```suggestion
        fflags = flags[:]
```